### ISSUE TITLE
Support for `SGE while browsing`

### ISSUE THREAD

**Issue by RoyRao2333** [Alex313031/Thorium-MacOS#5]
Please support for this feature which seems added in Chromium 116:

https://labs.google.com/search/experiment/2/springboard?is=ag

I've tried those examples and was not working. 😢

**Comment by gz83** [comment]
For an introduction and help about this function, see

https://support.google.com/websearch/answer/13551902?visit_id=638297390875406499-1969678396&p=search_labs&rd=1#zippy=

This feature is controlled by Google and has nothing to do with the browser

@RoyRao2333

**Comment by RoyRao2333** [comment]
@gz83 

Yes, I already turned them all on:

<img width="1793" alt="image" src="https://github.com/Alex313031/Thorium-MacOS/assets/31413093/abfce977-f365-4536-96e9-87b713685f71">

But still no effect...

**Comment by gz83** [comment]
This feature is only available on Google Search, and you may also need to join the waitlist and use a US network service to use this feature

@RoyRao2333

**Comment by RoyRao2333** [comment]
@gz83 

Thanks for the reply.

I'm using US network service and I'm not talking about `generative AI in search`. I'm talking about `SGE while browsing` which is the second one. 😊

Maybe you can try it out and test if you can make it work. Thanks!

**Comment by gz83** [comment]
Thank you for starting this issue. The information I obtained from you is just to confirm that you have fulfilled some prerequisites for using this feature as much as possible.

Since I don't live in the US, I'll ask Alex to try it out.

CC: @Alex313031

**Comment by RoyRao2333** [comment]
@gz83 

Thanks for the efforts! 

Not going to create another issue so I'm gonna ask here. After upgrading to latest Thorium, I found that tabs that I hovered on has two colors: An oval with darker background, and a trapezoidal with lighter background.

<img width="255" alt="image" src="https://github.com/Alex313031/Thorium-MacOS/assets/31413093/85f2a41c-44dd-4b78-b7a1-430f62793618">

I think this is difference from the previous version. Is this intentional? Thanks!
